core/src/Select/Select.less

```diff
@@ -9,22 +9,12 @@
   display: flex;
 
   &.show {
-    .selection {
```

Comment:
What's the reason for removing this style?

Reply:
@shrunyan  this was commented out code not in use, so  I cleaned it up